fundamental-concepts/data-structures-stdlib/index.json

```diff
@@ -0,0 +1,44 @@
+{
+  "title": "Data types and the DAML standard library",
+  "description": "Learn about records, sums, sets, and maps and other essential tools of the DAML standard library.",
+  "difficulty": "Beginner",
+  "time": "30 minutes",
+  "details": {
+    "assets": {
+        "host01": [
+        ]
+    },
+    "steps": [
+      {"text": "1_records.md"},
+      {"text": "2_sums.md"},
+      {"text": "3_type_synonyms.md"},
+      {"text": "4_imports.md"},
+      {"text": "5_stdlib.md"},
+      {"text": "6_lists.md"},
+      {"text": "7_sets.md"},
+      {"text": "8_maps.md"},
+      {"text": "9_optional.md"}
+    ],
+    "intro": {
+      "text": "0_intro.md",
+      "courseData": "init_background.sh"
+    },
+    "finish": {
+      "text": "99_finish.md"
+    }
+  },
+  "environment": {
+    "showide": false,
```

Comment:
Would be good to add some copy/pasting of the code, then we'd change this to 

"showide": true,

Reply:
most code is now copy/pasted and this set to `true`.